invenio_rdm_migrator/streams/runner.py

```diff
@@ -63,9 +71,11 @@ def run(self):
         for stream in self.streams:
             try:
                 stream.run()
-            finally:
+                # sucessfully finished stream run, now we can dump that stream cache
                 for name, cache in self.cache.items():
                     if name == "communities":  # FIXME: implement communities cache
                         continue
                     cache_file = self.cache_dir / f"{name}.json"
                     cache.dump(cache_file)
+            except Exception as exc:
+                print(exc)
```

Comment:
If there is no continuation, this has the same effect that no try/except block. No? (i.e. exception printed in the terminal)